src/components/modules/banners/usEoyAppeal/UsEoyAppeal.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import ContributionsTemplate from '../contributionsTemplate/ContributionsTemplate';
+import UsEoyAppealBody from './components/UsEoyAppealBody';
+import UsEoyAppealCloseButton from './components/UsEoyAppealCloseButton';
+import UsEoyAppealHeader from './components/UsEoyAppealHeader';
+import UsEoyAppealTicker from './components/UsEoyAppealTicker';
+import UsEoyAppealCta from './components/UsEoyAppealCta';
+import {
+    OPHAN_COMPONENT_EVENT_CONTRIBUTE_CLICK,
+    OPHAN_COMPONENT_EVENT_NOT_NOW_CLICK,
+    OPHAN_COMPONENT_EVENT_CLOSE_CLICK,
+} from './helpers/ophan';
+import withCloseable, { CloseableBannerProps } from '../hocs/withCloseable';
+
+const UsEoyAppealBanner: React.FC<CloseableBannerProps> = ({
+    isSupporter,
+    tickerSettings,
+    onClose,
+    submitComponentEvent,
+    tracking,
+    countryCode,
+}: CloseableBannerProps) => {
+    const onContributeClick = (): void =>
+        submitComponentEvent && submitComponentEvent(OPHAN_COMPONENT_EVENT_CONTRIBUTE_CLICK);
+
+    const onNotNowClick = (): void => {
+        submitComponentEvent && submitComponentEvent(OPHAN_COMPONENT_EVENT_NOT_NOW_CLICK);
+        onClose();
+    };
+
+    const onCloseClick = (): void => {
+        submitComponentEvent && submitComponentEvent(OPHAN_COMPONENT_EVENT_CLOSE_CLICK);
```

Comment:
do we normally track these events on banners?

Reply:
we did track these for the enviro banner - but that might have been the first time. I was kinda curious about close vs not now in dismissing the banner. Can remove though if you think that's better

Reply:
Yes it's good to track that 👍